Homework/homework1/task5.py

```diff
@@ -0,0 +1,20 @@
+"""
+Given a list of integers numbers "nums".
+
+You need to find a sub-array with length less equal to "k", with maximal sum.
+The written function should return the sum of this sub-array.
+Examples:
+    nums = [1, 3, -1, -3, 5, 3, 6, 7], k = 3
+    result = 16
+"""
+from typing import List
+
+
+def find_maximal_subarray_sum(nums: List[int], k: int) -> int:
+    if k > len(nums) or not nums or k == 0:
+        return 404
```

Comment:
Creative and compleatelly wrong.  When pass the wrong arguments, I never expect to get 404. 
I am ok both with zero and ValueError